config/envato.yml

```
@@ -64,6 +64,8 @@ finance-dev:
     - finance-dev
     - finance_infrastructure
     - elements-earnings
+    - identity
+    - identity-admin
```

`identity-infrastructure` will be shared for some time, so this probably belongs in both teams.

```suggestion
    - identity
    - identity-admin
    - identity-infrastructure
```

---

Thanks Pete! Addressed via 1cdcdee